dev/com.ibm.ws.logging_2_fat/fat/src/com/ibm/ws/logging/fat/MessageTraceFileNameTimedRolloverTest.java

```diff
@@ -0,0 +1,273 @@
+import static org.junit.Assert.*;
+
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+import org.junit.*;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.RemoteFile;
+
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.impl.LibertyServerFactory;
+
+@RunWith(FATRunner.class)
+public class MessageTraceFileNameTimedRolloverTest {
+
+    private static final String SERVER_NAME = "com.ibm.ws.logging.messagetracerollover";
+    private static final String[] EXPECTED_FAILURES = { "CWWKF0001E" };
+
+    private static LibertyServer server;
+    private static RemoteFile bootstrapFile = null;
+    private static Properties initialBootstrapProps = null;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        server = LibertyServerFactory.getLibertyServer(SERVER_NAME);
+
+        // Get the bootstrap.properties file and store original content
+        bootstrapFile = server.getServerBootstrapPropertiesFile();
+        FileInputStream in = getFileInputStreamForRemoteFile(bootstrapFile);
+        initialBootstrapProps = loadProperties(in);
+
+        // Preserve the original server configuration
+        server.saveServerConfiguration();
+    }
+
+    @After
+    public void cleanupAfterEachTest() throws Exception {
+        if (server != null && server.isStarted()) {
+            server.stopServer(EXPECTED_FAILURES);
+        }
+
+        // Restore the initial contents of bootstrap.properties
+        if (bootstrapFile != null && initialBootstrapProps != null) {
+            FileOutputStream out = getFileOutputStreamForRemoteFile(bootstrapFile, false);
+            writeProperties(initialBootstrapProps, out);
+        }
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        if (server != null && server.isStarted()) {
+            server.stopServer(EXPECTED_FAILURES);
+        }
+    }
+
+    /**
+     * Test 1: Configure TimedLogRollover (in server.env) and set traceFileName=stdout in bootstrap.properties,
+     * verify if trace.log is not created, only message.log is rolled over.
+     */
+    @Test
+    public void testTraceFileNameStdoutBootstrapWithTimedRollover() throws Exception {
+        // Simple - just start server (bootstrap.properties already has traceFileName=stdout)
+        server.startServer();
+        server.waitForStringInLog("CWWKF0011I");
+
+        boolean traceLogExists = server.fileExistsInLibertyServerRoot("logs/trace.log");
+        assertFalse("REQUIREMENT: trace.log should NOT be created when traceFileName=stdout", traceLogExists);
+
+        boolean messageLogExists = server.fileExistsInLibertyServerRoot("logs/messages.log") ||
+                                   server.fileExistsInLibertyServerRoot("logs/message.log");
+        assertTrue("message log should exist", messageLogExists);
```

Comment:
According to the test description, this test should test if the messages.log gets rolled over, as well.